### PR TITLE
Define ControllerDocument class.

### PR DESCRIPTION
**Pull request by msporny**
This PR is an attempt to address issue https://github.com/w3c/vc-data-integrity/issues/322 by defining a ControllerDocument class. 

/cc @filip26

### CODE REVIEW

**Review by dlongley — APPROVED** [review]

**Comment by iherman** [comment]
The PR is incomplete, because the diagram must be updated as well and. It is also a question whether the domain of the "controller" property shouldn't be specified as a union (ie, logical 'or') for a controller document class and the verification method class.

@msporny I think I should take it over, because I have more practice in creating/changing the diagram. I will clone your branch and take it from there if you are fine with it.

**Comment by msporny** [comment]
> It is also a question whether the domain of the "controller" property shouldn't be specified as a union (ie, logical 'or') for a controller document class and the verification method class.

I don't think we should limit the domain just yet... controller might make sense in many more use cases.

> @msporny I think I should take it over, because I have more practice in creating/changing the diagram. I will clone your branch and take it from there if you are fine with it.

Yes, please take it over @iherman ... and thank you! :)

**Comment by iherman** [comment]
> > It is also a question whether the domain of the "controller" property shouldn't be specified as a union (ie, logical 'or') for a controller document class and the verification method class.
> 
> I don't think we should limit the domain just yet... controller might make sense in many more use cases.
> 
> > @msporny I think I should take it over, because I have more practice in creating/changing the diagram. I will clone your branch and take it from there if you are fine with it.
> 
> Yes, please take it over @iherman ... and thank you! :)

Done.

**Review by iherman — APPROVED** [review]

File: vocab/security/template.html [hunk 1]
```diff
@@ -270,7 +270,9 @@ <h2>Diagram description</h2>
         </p>
         <p>
           Each of these sections has an ellipse at the top, labeled as "Proof", "VerificationMethod", 
-          and "VerificationRelationship", respectively.
+          and "VerificationRelationship", respectively. Additionally, the Verification Method section also
+          includes an additional ellipse, labeled as "ControllerDocument", side-by-side with the ellipse
+          labeled as "VerificationMethod".
```
Comment (TallTed) [inline vocab/security/template.html]:
```suggestion
          and "VerificationRelationship", respectively. The Verification Method section
          includes an additional ellipse, labeled as "ControllerDocument", side-by-side with the ellipse
          labeled as "VerificationMethod".
```

File: vocab/security/template.html [hunk 2]
```diff
@@ -304,16 +306,17 @@ <h3>Verification Method Section</h3>
...
           <p>
             The left side of this section contains a column of three labeled boxes, all styled as "Property". 
-            The labels, from top to bottom, are "expires", "controller", and "revoked". 
+            The labels, from top to bottom, are "controller", "expires", and "revoked".
             Each of these is connected to the "VerificationMethod" ellipse, with a line styled as "Domain Of".
-            The "expires" "Property" box is also connected to the "Proof" ellipse (in the Proof section), 
-            with a line styled as "Domain Of".
+            The "controller" "Property" box is also connected to the "ControllerDocument" ellipse, while
+            the "expires" "Property" box is also connected to the "Proof" ellipse (in the Proof section);
+            both these extra connections are also with a line styled as "Domain Of".
```
Comment (TallTed) [inline vocab/security/template.html]:
```suggestion
            both of these extra connecting lines are styled as "Domain Of".
```

**Review by TallTed — CHANGES_REQUESTED** [review]
The redundant department of redundancy department thanks you for your support.

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**